### ISSUE TITLE
Not working in updated Loco

### ISSUE THREAD

**Issue by jitendrarout33** [dipendrapkrl/TriviaHelper#62]
Hii sie ..The app is awesome as compare to any other helper bt its not working in updated loco.So please bring an update and fix this issue thank you.

**Comment by ashok-bansal** [comment]
Yes, loco has set some flag to disable screen capture. Do something to bypass that flag.

**Comment by akashchaurasiya** [comment]
How, can I solve this problem? 
Can you tell me?

**Comment by akashchaurasiya** [comment]
I also didn't able to take screenshot.

**Comment by akashchaurasiya** [comment]
How, can I remove this flag.
